x/vm/keeper/grpc_query.go

```diff
@@ -641,6 +647,44 @@ func (k *Keeper) traceTx(
 		overrides = traceConfig.Overrides.EthereumConfig(types.GetEthChainConfig().ChainID)
 	}
 
+	// Handle access list tracer specifically
+	if traceConfig.Tracer == types.TracerAccessList {
```

Comment:
this should be its own function since it is some separate specific logic with no side effects